src/client/java/teammates/client/scripts/DataMigrationForResponseRate.java

```diff
@@ -51,27 +50,18 @@ private void updateRespondentsForAllSessions() {
                          ? getFeedbackSessionsWithZeroResponseRate()
                          : fsDb.getAllFeedbackSessions();
 
-        for (FeedbackSessionAttributes session : feedbackSessions) {
-            updateRespondentsForSession(session.getFeedbackSessionName(), session.getCourseId());
-        }
+        feedbackSessions.forEach((session) -> updateRespondentsForSession(
+                session.getFeedbackSessionName(), session.getCourseId()));
     }
 
     private List<FeedbackSessionAttributes> getFeedbackSessionsWithZeroResponseRate() {
         @SuppressWarnings("deprecation")
         List<FeedbackSessionAttributes> feedbackSessions = fsDb.getAllFeedbackSessions();
 
-        List<FeedbackSessionAttributes> feedbackSessionsWithNoRespondents = new ArrayList<>();
-
-        for (FeedbackSessionAttributes feedbackSession : feedbackSessions) {
-            if (feedbackSession.getRespondingStudentList().size() != 0
-                    || feedbackSession.getRespondingInstructorList().size() != 0) {
-                continue;
-            }
-
-            feedbackSessionsWithNoRespondents.add(feedbackSession);
-        }
+        feedbackSessions.removeIf((feedbackSession) -> feedbackSession.getRespondingStudentList().size() != 0
+                                                 || feedbackSession.getRespondingInstructorList().size() != 0);
```

Comment:
This is good since it not only reduces the number of lines of code but also makes the program flow simpler.